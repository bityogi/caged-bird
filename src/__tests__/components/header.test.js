jest.mock('util/axiosClient');

import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
import { MemoryRouter } from 'react-router';

import HeaderComponent from 'components/header';
import store from 'store';

describe('Header Component', () => {
  it('renders correctly', () => {
    const rendered = renderer.create(
      <Provider store={store}>
        <MuiThemeProvider>
          <MemoryRouter initialEntries={['/', '/landing']}>
            <HeaderComponent />
          </MemoryRouter>
        </MuiThemeProvider>
      </Provider>

    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });

});
