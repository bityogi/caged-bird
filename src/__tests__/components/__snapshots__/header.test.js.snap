// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders correctly 1`] = `
<div>
  <div
    className={undefined}
    style={
      Object {
        "WebkitTapHighlightColor": "rgba(0,0,0,0)",
        "backgroundColor": "#00838f",
        "boxSizing": "border-box",
        "display": "flex",
        "height": 56,
        "justifyContent": "space-between",
        "muiPrepared": true,
        "padding": "0px 24px",
      }
    }
  >
    <div
      className={undefined}
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "justifyContent": "space-between",
          "marginLeft": -24,
          "marginRight": undefined,
          "muiPrepared": true,
          "position": "relative",
        }
      }
    >
      <span
        onClick={[Function]}
        style={
          Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "cursor": "pointer",
            "fontFamily": "Roboto, sans-serif",
            "fontSize": "1.8em",
            "lineHeight": "56px",
            "marginLeft": 25,
            "muiPrepared": true,
            "overflow": "hidden",
            "paddingRight": 16,
            "position": "relative",
            "textOverflow": "ellipsis",
            "whiteSpace": "nowrap",
          }
        }
      >
        Caged Bird
      </span>
      <span
        className={undefined}
        style={
          Object {
            "backgroundColor": "rgba(0, 0, 0, 0.175)",
            "color": "rgba(0, 0, 0, 0.4)",
            "display": "block",
            "height": 32,
            "lineHeight": "56px",
            "margin": 20,
            "marginLeft": 24,
            "muiPrepared": true,
            "width": "2px",
          }
        }
      />
    </div>
    <div
      className={undefined}
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "justifyContent": "space-between",
          "marginLeft": undefined,
          "marginRight": undefined,
          "muiPrepared": true,
          "position": "relative",
        }
      }
    >
      <img
        alt="Caged-Bird"
        height={48}
        src="LOGO_WHITE.png"
        style={
          Object {
            "marginBottom": 3,
            "marginRight": 50,
            "marginTop": 3,
          }
        }
        width={120}
      />
    </div>
  </div>
</div>
`;
